driver/image_adjust_brightness_driver.hpp

```diff
@@ -0,0 +1,301 @@
+        std::cerr << "Error copying buffer to GPU with size " << input_gpu->GetSize() << std::endl;
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ImageAdjustBrightnessDriver<Tgpu, Tref>::RunForwardGPU()
+{
+    float kernel_total_time = 0;
+    float kernel_first_time = 0;
+
+    Timer t;
+    START_TIME
+
+    for(int i = 0; i < inflags.GetValueInt("iter"); i++)
+    {
+        miopenImageAdjustBrightness(GetHandle(),
+                                    inputTensorDesc,
+                                    outputTensorDesc,
+                                    input_gpu->GetMem(),
+                                    output_gpu->GetMem(),
+                                    brightness_factor);
+
+        float time = 0.0;
+        miopenGetKernelTime(GetHandle(), &time);
+        kernel_total_time += time;
+        if(i == 0)
+            kernel_first_time = time;
+    }
+
+    if(inflags.GetValueInt("time") == 1)
+    {
+        STOP_TIME
+        int iter = inflags.GetValueInt("iter");
+
+        if(WALL_CLOCK)
+            std::cout << "Image Adjust Brightness Time Elapsed: " << kernel_total_time << " ms"
+                      << std::endl;
+
+        float kernel_avg_time =
+            iter > 1 ? (kernel_total_time - kernel_first_time) / (iter - 1) : kernel_total_time;
+
+        std::cout << "Image Adjust Brightness Kernel Time Elapsed: " << kernel_avg_time << " ms"
+                  << std::endl;
+    }
+
+    if(output_gpu->FromGPU(GetStream(), out_host.data()) != miopenStatusSuccess)
+    {
+        std::cerr << "Error copying buffer from GPU with size " << output_gpu->GetSize()
+                  << std::endl;
+        return -1;
+    }
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ImageAdjustBrightnessDriver<Tgpu, Tref>::RunForwardCPU()
+{
+    mloImageAdjustBrightnessRunHost(in_host.data(),
+                                    out_ref.data(),
+                                    miopen::deref(inputTensorDesc),
+                                    miopen::deref(outputTensorDesc),
+                                    brightness_factor);
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ImageAdjustBrightnessDriver<Tgpu, Tref>::VerifyForward()
+{
+    RunForwardCPU();
+
+    auto threashold = std::numeric_limits<Tgpu>::epsilon();
+    auto error      = miopen::rms_range(out_ref, out_host);
+
+    if(!std::isfinite(error) || error > threashold)
+    {
+        std::cout << "Forward Image Adjust Brightness FAILED: " << error << std::endl;
+    }
+    else
+    {
+        std::cout << "Forward Image Adjust Brightness Verifies on CPU and GPU (" << error << ')'
+                  << std::endl;
+    }
+
+    return miopenStatusSuccess;
+}
+
+template <typename Tgpu, typename Tref>
+int ImageAdjustBrightnessDriver<Tgpu, Tref>::RunBackwardGPU()
+{
+    // Does not exist
+    return miopenStatusSuccess;
```

Comment:
Did you try `miopenStatusNotImplemented` ?

Reply:
It has to be that way (check the Sum driver for a similar case where backwards doesn't exist)